static/app/components/devtoolbar/components/issues/issuesPanel.tsx

```diff
@@ -13,7 +16,20 @@ import PanelLayout from '../panelLayout';
 import useInfiniteIssuesList from './useInfiniteIssuesList';
 
 export default function IssuesPanel() {
-  const transactionName = useCurrentTransactionName();
+  const {scope, client} = useScopeAndClient();
+  const [transactionName, setTransactionName] = useState(
+    scope?.getScopeData().transactionName
```

Comment:
need to call `toSearchTerm()` in all the places... probably better to have a `const transactionName = scope?.getScopeData().transactionName` separate from a `const transactionSearchTerm = toSearchTerm(transactionName)` local variable separate.